tests/playwright/specs/wikipedia.js

```diff
@@ -0,0 +1,14 @@
+let assert = require( 'assert' );
+const { chromium } = require( 'playwright' );
+
+( async () => {
+	let browser, page;
```

Comment:
why are we initializing browser and page separately?
Please use const wherever possible